docs/pages/components/SearchField.mdx

```diff
@@ -96,6 +106,12 @@ By default, SearchField is wrapped in a `<form role="search">` to provide better
       description:
         'If true, the label and input will be rendered in a `<form>` tag. If false, the label and input will be rendered in a `<div>` tag.',
       defaultValue: 'true'
+    },
+    {
+      name: 'trailingChildren',
+      type: ['ReactNode, string'],
```

Comment:
[`ReactElement`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L318-L325) is an element/component type where `ReactNode` covers a wider range. `ReactNode` also already includes string so we should just be able to use a single type here.

```suggestion
      type: 'ReactNode',
```